examples/scripts/ppo-sentiment.py

```diff
@@ -41,9 +41,12 @@
 # We first define the configuration of the experiment, defining the model, the dataset,
 # the training parameters, and the PPO parameters.
 # Check the default arguments in the `PPOConfig` class for more details.
+# If you want to log with tensorboard, add the kwarg
+# `accelerator_kwargs={"logging_dir": PATH_TO_LOGS}` to the PPOConfig.
 config = PPOConfig(
     model_name="lvwerra/gpt2-imdb",
     learning_rate=1.41e-5,
+    log_with="wandb",
```

Comment:
if we remove wandb as a main dependency i think we should not use it as a default. otherwise it will fail if you use the ppo trainer out of the box. what do you think?

Reply:
yes makes sense!

Reply:
Should be now addressed in [d3f9231](https://github.com/lvwerra/trl/pull/92/commits/d3f923152cf670296dbed2223ef0cbdd78f760c8) !